README.Rmd

```diff
@@ -14,66 +17,87 @@ knitr::opts_chunk$set(
 ```
 
 <!-- badges: start -->
-[![CRAN status](https://www.r-pkg.org/badges/version/vol2birdR)](https://cran.r-project.org/package=vol2birdR)
+
+[![CRAN
+status](https://www.r-pkg.org/badges/version/vol2birdR)](https://cran.r-project.org/package=vol2birdR)
 [![R-CMD-check](https://github.com/adokter/vol2birdR/workflows/R-CMD-check/badge.svg)](https://github.com/adokter/vol2birdR/actions)
+
 <!-- badges: end -->
 
 # vol2birdR
-'**vol2birdR**' is an 'R' package for the 'vol2bird' algorithm for calculating vertical profiles of birds and other biological scatterers from weather radar data. 
 
-It also provides an 'R' interface to the 'MistNet' convolutional neural network for precipitation segmentation, installing PyTorch libraries and model.
+'**vol2birdR**' is an 'R' package for the 'vol2bird' algorithm for
+calculating vertical profiles of birds and other biological scatterers
+from weather radar data.
 
-'**vol2birdR**' can be used as a stand-alone package, but we recommend [bioRad](https://adriaandokter.com/bioRad/) as the primary user interface, with '**vol2birdR**' acting as a dependency of [bioRad](https://adriaandokter.com/bioRad/).
+It also provides an 'R' interface to the 'MistNet' convolutional neural
+network for precipitation segmentation, installing PyTorch libraries and
+model.
+
+'**vol2birdR**' can be used as a stand-alone package, but we recommend
+[bioRad](https://adriaandokter.com/bioRad/) as the primary user
+interface, with '**vol2birdR**' acting as a dependency of
+[bioRad](https://adriaandokter.com/bioRad/).
 
 # Install
```

Comment:
I think we need a brief description of mistnet installation as well, I'll add something to this PR